code/__DEFINES/~skyrat_defines/vore.dm

```diff
@@ -0,0 +1,75 @@
+#define VORE_SAVEFILE_VERSION			1 //UPDATE THIS AND CREATE SANITY CHECKS IF YOU MAKE MAJOR CHANGES
+
+//TOGGLES - EVERYTHING YOU NEED TO UPDATE FOR NEW THINGS IS IN THIS SECTION OR IN THE switch_toggles() PROC IN vore_prefs.dm (although if creating a new section, update sanitize_vars() in vore_prefs.dm)
+
+#define VORE_MECHANICS_TOGGLES			"mechanics"
```

Comment:
Remove all occurrences of this formatting too, multiple tabs. There should only be a space between the define and the value.